_posts/2016-xx-xx-release-0.16.md

```diff
@@ -126,42 +126,49 @@ MDAnalysis typically reads trajectories from files on-demand, so that it can eff
 The MemoryReader works with numpy arrays, using the same format as that used by for instance `DCDReader.timeseries()`. You can create a Universe directly from such an array:
 
 ```python
+import numpy as np
 from MDAnalysis import Universe
-from MDAnalysisTests.datafiles import DCD, PDB_small
+from MDAnalysisTests.datafiles import DCD, PSF
 from MDAnalysis.coordinates.memory import MemoryReader
 
 # Create a Universe using a DCD reader
-universe = Universe(PDB_small, DCD)
+universe = Universe(PSF, DCD)
 
-# Extract coordinates
-coordinates = universe.trajectory.timeseries()
+# Create a numpy array with random coordinates (100 frames) for the same topology
+coordinates = np.random.uniform(size=(universe.atoms.n_atoms, 100, 3)).cumsum(0)
 
 # Create a new Universe directly from these coordinates
-# using the MemoryReader
-universe2 = Universe(PDB_small, coordinates,
-                     format=MemoryReader)
+universe2 = Universe(PSF, coordinates, format=MemoryReader)
 ```
 
-The MemoryReader will work just as any other reader. In particular, you can iterate over it as usual, or use the `.timeseries()` method to retrieve a reference to the raw array in any format:
+The MemoryReader will work just as any other reader. In particular, you can iterate over it as usual, or use the `.timeseries()` method to retrieve a reference to the raw array in any order of the dimensions ('fac' means ('frames','atoms','coordinates')):
```

Comment:
@orbeckst I've removed the "order of the dimensions ('fac' means ('frames','atoms','coordinates'))" as you suggested.